Sources/UIAlertControllerView.swift

```diff
@@ -43,48 +57,100 @@ class UIAlertControllerView: UIView {
             return button
         }
 
+        clipsToBounds = true
         backgroundColor = .white
         layer.cornerRadius = 3
 
-        header.sizeToFit()
-        header.frame.height += verticalPadding
-        header.layer.contentsGravity = .top
         addSubview(header)
+        text.map { addSubview($0) }
+        buttons.forEach { addSubview($0) }
+
+        subviewWidth = {
+            let minWidth: CGFloat = 300
+            let maxReadableWidth: CGFloat = 540
```

Comment:
What is this based on? Is it related to the default font size vs the line width? I would love to see this as a product of some other known quantities rather than something hard coded. I don't consider this to be a blocker but it would be great if we could get more insight in the code as to why this value is what it is

Reply:
@mouselangelo suggested this value to have something to limit the width of the text. I think ideally we would come up with a value that's based on the recommended line length of 50-75 characters ([src](https://baymard.com/blog/line-length-readability))

Reply:
Ok understood! Sounds like this would vary based on the font size of the labels then? AFAIK devs can adjust the font size of these labels so it would be great if you could at least add a comment stating that this value assumes the default font size of 12 (?) We don't have to implement this "perfectly" but it would be valuable to clarify our assumptions in the code

Reply:
Yes - this is very much tied to the font-size and the line-height and actually also the font itself.